### DISCUSSION TITLE
Uncertainty of the percentage of neutrons moving backwards after scattering

### DISCUSSION THREAD

**Discussion by XuShuqi7** [mctools/ncrystal#141] [Q&A] [answered]
Hi @tkittel,

I am using NCrystal to obtain the percentage of neutrons moving backwards after scattering with graphite for incident neutron energy fixed at 1 meV. What I tried is

```
graphite = NC.createScatter('C_sg194_pyrolytic_graphite.ncmat;temp=300K')
Ep, mu = graphite.sampleScatterIsotropic(0.001, repeat=1000000)
```

From $\mu$ we can get the percentage of neutrons moving backwards $B$. Since the sampling of neutron scattering is a random process, I would like to know if we can get the uncertainty of $B$.

Thanks a lot!
Shuqi

**Comment by tkittel** [comment, marked as answer]
Not sure if this is a question related to NCrystal or a general statistics question? If you shoot $N_\text{tot}$ neutrons and observe $N_\text{back}$ neutrons moving backwards, then the statistical uncertainty on the fraction $N_\text{back}/N_\text{tot}$ should be $\sqrt{N_\text{back}}/N_\text{tot}$, no? Assuming adequate statistics and all that.

And of course, this is a number for a single forced scattering, which is not the same as you might get for a real stepping monte carlo in a finite volume.

**Reply by XuShuqi7** [reply]
Thanks a lot @tkittel ! The statistical uncertainty is what I am looking for.
